ui.tests/test-module/specs/datepicker/datepicker.runtime.spec.js

```diff
@@ -275,7 +276,7 @@ describe("Form Runtime with Date Picker", () => {
 
     it("Test custom error message when incorrect date format is entered", () => {
         const [datePicker7, datePicker7FieldView] = Object.entries(formContainer._fields)[6];
-        const incorrectInputs = ["adfasdfa", "29/2/2023", "32/1/2023", "1/32/23", "1-1-2023"];
```

Comment:
@vdua What should be the expected behavior if incorrect value is entered, today the parse function accepts these value even if the format is d/m/yyyy ?